packages/mobile/src/screens/explore-screen/SearchExploreScreen.tsx

```diff
@@ -170,11 +186,20 @@ export const SearchExploreScreen = () => {
                 <Flex>
                   <TextInput
                     label='Search'
+                    autoFocus={autoFocus}
                     placeholder={messages.searchPlaceholder}
                     size={TextInputSize.SMALL}
                     startIcon={IconSearch}
                     onChangeText={setSearchInput}
                     value={searchInput}
+                    endIcon={() => (
```

Comment:
to maintain the contract, you can use the incomping props which include iconSize and color:

```
endIcon={(props) => (
  <IconButton icon={IconCloseAlt} onPress={handleClearSearch} hitSlop={10} {...props} />